src/main/java/au/org/ala/biocache/dto/DownloadDoiDTO.java

```diff
@@ -14,13 +14,20 @@
  ***************************************************************************/
 package au.org.ala.biocache.dto;
 
+import au.org.ala.biocache.service.DoiService;
+import org.apache.log4j.Logger;
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 
 /**
  * Encapsulates parameters need it to mint a DOI from the offline download functionality
  */
 public class DownloadDoiDTO {
+    public final static List<String> validDisplayTemplates = Arrays.asList(DoiService.DISPLAY_TEMPLATE_BIOCACHE, DoiService.DISPLAY_TEMPLATE_CSDM);
```

Comment:
Would prefer if this wasn't hardcoded, as biocache-service has a number of non-ALA users who may want their own, or who won't consider CSDM to be a display template that they want to support.

Reply:
Unfortunately, biocache-service existing architecture does not permit this flexibility. Adding it is out of scope for CSDM.

Reply:
This is a new feature that you are adding. How it is implemented is inside of the scope of this pull request.

Reply:
Let us have a phone conversation next week.